app/javascript/Components/assignment_summary_table.jsx

```diff
@@ -30,16 +48,145 @@ export class AssignmentSummaryTable extends React.Component {
     this.fetchData();
   }
 
-  toggleShowInactiveGroups = showInactiveGroups => {
-    let filtered = this.state.filtered.filter(group => {
-      group.id !== "inactive";
+  getColumns = () => {
+    const columnHelper = createColumnHelper();
+
+    const expanderColumn = columnHelper.display({
+      id: "expander",
+      header: () => null,
+      size: 32,
+      cell: ({row}) => {
+        return row.getCanExpand() ? (
+          <button
```

Comment:
This is pretty good, but can be improved:

1. All styles should go into a CSS file, not be set here
2. The spread operator (`...`) is unnecessary. Just do `onClick={...}` etc. to pass attributes to the `button`.
3. Rather than using separate unicode characters, take a look at how the original did it:
    - component: https://github.com/TanStack/table/blob/v6/src/defaultProps.js#L237-L239
    - CSS: https://github.com/rdubigny/react-table-v6/blob/8cffee96fad788484edf512aa8fe4fab176e803b/src/index.styl#L155-L175

    You should be able to copy the styling into our CSS file as well to get the triangle icon to show up.